linera-base/src/identifiers.rs

```diff
@@ -29,12 +32,30 @@ use crate::{
 
 /// An account owner.
 #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, WitLoad, WitStore, WitType)]
-#[cfg_attr(with_testing, derive(test_strategy::Arbitrary))]
 pub enum AccountOwner {
     /// Short addresses reserved for the protocol.
     Reserved(u8),
     /// 32-byte account address.
     Address32(CryptoHash),
+    /// 20-byte account EVM-compatible address.
+    Address20(Address),
+}
+
+#[cfg(with_testing)]
+impl Arbitrary for AccountOwner {
```

Comment:
Had to implement this one manually as there was none for `alloy_primitives::Address` implemented and Rust prohibits from implementing foreign traits for foreign types.